Example/CardParts/CardPartOrientedViewCardController.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  CardPartOrientedViewCardController.swift
+//  CardParts_Example
+//
+//  Created by Tumer, Deniz on 6/13/18.
+//  Copyright © 2018 CocoaPods. All rights reserved.
```

Comment:
I've been seeing it do this lately with mine as well... should change to `© 2018 Intuit`

Reply:
fixed these in most recent commit